set.mm

```diff
@@ -485890,6 +485921,101 @@ commutative monoid (=vectors) together with a semiring (=scalars) and a
+        ( vf vs cvv wcel cv wss csdrg cfv crab cint cfldgen wceq co elexd fvexi
+        cdr cbs a1i ssexd sdrgid syl wb sseq2 adantl rspcedvd intexrab sylib wa
+        simpl fveq2d simpr sseq1d rabeqbidv inteqd df-fldgen ovmpoga syl3anc
+        wrex ) ADKLCKLCEMZNZEDOPZQZRZKLZDCSUAVKTADUDGUBACBKBKLABDUEFUCUFHUGAVHE
+        VIVFVLAVHCBNZEBVIADUDLBVILGBDFUHUIVGBTVHVMUJAVGBCUKULHUMVHEVIUNUOIJDCKK
+        JMZVGNZEIMZOPZQZRVKSKVPDTZVNCTZUPZVRVJWAVOVHEVQVIWAVPDOVSVTUQURWAVNCVGV
+        SVTUSUTVAVBIJEVCVDVE $.
+
+      $( A generated subfield is a sub-division-ring.  (Contributed by Thierry
+         Arnoux, 11-Jan-2025.) $)
+      fldgensdrg $p |- ( ph -> ( F fldGen S ) e. ( SubDRing ` F ) ) $=
+        ( va vx co cv wss cfv cdr wcel cress crg wa issdrg syl crab cint csubrg
+        cfldgen csdrg fldgenval drngringd eqid sseq2 elrab biimpi adantl simpld
+        cur simp2bi ex ssrdv sdrgid elrabd simp3bi subdrgint intss1 wi subrg1cl
+        ne0d wral ad2antlr ralrimiva elintrab sylibr issubrg syl22anc syl3anbrc
+        fvex biimpri eqeltrd ) ADCUDJCHKZLZHDUEMZUAZUBZVSABCDHEFGUFADNOZWADUCMZ
+        OZDWAPJZNOWAVSOFADQOZWEQOZWABLZDUNMZWAOZWDADFUGAWEADVTWEIWEUHFAIVTWCAIK
+        ZVTOZWKWCOZAWLRZWKVSOZWMWNWOCWKLZWLWOWPRZAWLWQVRWPHWKVSVQWKCUIUJUKULUMZ
+        WOWBWMDWKPJNOZDWKSZUOTUPUQAVTBAVRCBLHBVSVQBCUIAWBBVSOFBDEURTGUSZVEWNWOW
+        SWRWOWBWMWSWTUTTVAZUGABVTOWHXABVTVBTAVRWIVQOZVCZHVSVFWJAXDHVSAVQVSOZRVR
+        XCXEXCAVRXEVQWCOZXCXEWBXFDVQPJNODVQSUOVQDWIWIUHZVDTVGUPVHVRHWIVSDUNVNVI
+        VJWDWFWGRWHWJRRWABDWIEXGVKVOVLXBDWASVMVP $.
+    $}
+
+    $( The subfield of a field ` F ` generated by the whole base of ` F ` is
+       ` F ` itself.  (Contributed by Thierry Arnoux, 11-Jan-2025.) $)
+    fldgenid $p |- ( ph -> ( F fldGen B ) = B ) $=
+      ( va cfldgen co cv wss csdrg cfv crab cint ssidd fldgenval wcel sseq2 cdr
+      syl sdrgid elrabd intss1 wi wral wa ralrimiva ssintrab sylibr eqssd eqtrd
+      idd ) ACBGHBFIZJZFCKLZMZNZBABBCFDEABOZPAUQBABUPQUQBJAUNBBJFBUOUMBBRACSQBU
+      OQEBCDUATURUBBUPUCTAUNUNUDZFUOUEBUQJAUSFUOAUMUOQUFUNULUGUNFBUOUHUIUJUK $.
+  $}
+
+  ${
+    fldgenfld.1 $e |- B = ( Base ` F ) $.
+    fldgenfld.2 $e |- ( ph -> F e. Field ) $.
+    fldgenfld.3 $e |- ( ph -> S C_ B ) $.
+    $( A generated subfield is a field.  (Contributed by Thierry Arnoux,
+       11-Jan-2025.) $)
+    fldgenfld $p |- ( ph -> ( F |`s ( F fldGen S ) ) e. Field ) $=
+      ( cfield wcel cfldgen co csdrg cfv cress cdr ccrg wa isfld sylib simpld
+      fldgensdrg fldsdrgfld syl2anc ) ADHIZDCJKZDLMIDUENKHIFABCDEADOIZDPIZAUDUF
+      UGQFDRSTGUAUEDUBUC $.
+  $}
+
+  ${
+    $d .1. a $.  $d B a $.  $d R a $.  $d a ph $.
+    primefldgen1.b $e |- B = ( Base ` R ) $.
+    primefldgen1.1 $e |- .1. = ( 1r ` R ) $.
+    primefldgen1.r $e |- ( ph -> R e. DivRing ) $.
+    $( The prime field is the subfield generated by the multiplicative identity
+       element.  (Contributed by Thierry Arnoux, 11-Jan-2025.) $)
```

Comment:
```suggestion
    $( The prime field of a ring is the subfield generated by the multiplicative identity
       element.  (Contributed by Thierry Arnoux, 11-Jan-2025.) $)
```
Same as above: it is a "prime division ring", so maybe a remark in the comment: 'In general, we should write "prime division ring", but since most later usages are in the case where the ambient ring is commutative, we keep the term "prime field".'

Reply:
Done!